### PR TITLE
Nullable and defaults

### PR DESCRIPTION
**Pull request by Synicix**
- Redesign the way the input blocks and generated for datetime, timestamp, and date to handle default value and current state.
- Added support for reseting to null or default for insert and update (#48) PR #93
- Added support to render default values if exist for insert (#95) PR #93

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 - Support for double data type (#72) PR #86
 - Added search functionailty for table list similar to schema. (#73) PR #88
 - Added input field for date/datetime/time/timestamp copy over for insert and update. (#47) PR #87
+- Added support for reseting to null or default for insert and update (#48) PR #93
+- Added support to render default values if exist for insert (95) PR #93
```
Comment (guzman-raphael) [inline CHANGELOG.md:11]:
```suggestion
- Added support to render default values if exist for insert (#95) PR #93
```

File: src/Components/MainTableView/DataStorageClasses/SecondaryTableAttribute.tsx [hunk 2]
```diff
@@ -8,37 +8,48 @@ import {faRedoAlt} from '@fortawesome/free-solid-svg-icons'
...
     this.nullable = nullable;
-    this.defaultValue = defaultValue;
+    this.defaultValue = defaultValue === null || defaultValue === 'null' ? undefined : defaultValue;
   }
 
   static getAttributeLabelBlock(secondaryTableAttribute: SecondaryTableAttribute, resetToNullCallback: any) {
     const typeString = super.getTypeString(secondaryTableAttribute);
+    var resetButtonText = 'nullable';
+
+    // I don't think this detects is correctly
```
Comment (guzman-raphael) [inline src/Components/MainTableView/DataStorageClasses/SecondaryTableAttribute.tsx:31]:
```suggestion
    // I don't think this detects it correctly
```

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
Minor doc nits.

File: src/Components/MainTableView/DataStorageClasses/SecondaryTableAttribute.tsx (hunk 2 above)
Comment (Synicix) [reply src/Components/MainTableView/DataStorageClasses/SecondaryTableAttribute.tsx:31]: I fix this one, at least with the test cases I tried

**Review by guzman-raphael — APPROVED** [review]

**Pull request merged**